### ISSUE TITLE
Mobile Bug

### ISSUE THREAD

**Issue by phuree100** [whs/score#3]
The Version 2.0 and 3.0 not support mobile but the mobile in master version is **very bug** thanks for supporting

**Comment by whs** [comment]
The web design was created back during the turn of the technology. At that time the top of the line iPhone was iPhone 4 or 5, and most people used Blackberry or Symbian. There was an idea to use [jQuery Mobile](https://jquerymobile.com/) to develop the mobile version, but one of the most used feature people used is to open multiple dialogues at once to compare score of multiple people or multiple days which is lost in the mobile version. I thought the mobile UI couldn't top the desktop one, so I removed it and add the .txt file documenting the format hoping that future people might develop native apps or other stuff to interact with it.

For v3 I was porting the current design as-is to test the rewrite that it is working as intended. At the same time I contacted my UX friend to help with the redesign (which is why the popup didn't work properly - I don't think the final product will have it, and it's not worth the time to reimplement it temporarily). I still haven't got one so the project is on hiatus.

Anyway this is not a bug so I'm closing.
